lang/BUILD

```diff
@@ -0,0 +1,7 @@
+licenses(["notice"])  # Apache 2.0
+
+java_binary(
```

Comment:
do you prefer not using //lang/java/src/main/java/com/google/devtools/build/bfg:JavaSourceFileParserCli ?